src/DatePicker/Date.elm

```diff
@@ -0,0 +1,153 @@
+module DatePicker.Date exposing
+    ( YearRange(..)
+    , changeYear
+    , formatDay
+    , formatMonth
+    , initDate
+    , weekdayToInterval
+    , yearRange
+    )
+
+import Date exposing (Date, Interval(..), Unit(..), day, month, year)
+import Time exposing (Month(..), Weekday(..))
+
+
+type alias Year =
+    Int
+
+
+type alias Day =
+    Int
+
+
+type YearRange
+    = Off
+    | MoreOrLess Int
+    | Between Year Year
+    | From Year
+    | To Year
+
+
+initDate : Date
+initDate =
+    Date.fromCalendarDate 1992 May 31
+
+
+formatDay : Date.Weekday -> String
```

Comment:
I'm using `ryannhg/date-format` package somewhere in this project to format timestamps. Maybe you could use it instead of writing your own functions?

Reply:
It's not my code. I copied DatePicker code due because otherwise compiler crashed (when it was used as a package). This workaround is described in the issue I link to from one of the commit messages (one that adds ^ file).

Reply:
Ok.